client/my-sites/plugins/controller.js

```diff
@@ -144,6 +145,35 @@ function renderProvisionPlugins() {
 }
 
 controller = {
+	validateFilters: function( filter, context, next ) {
+		const wpcomFilters = [ 'standard', 'business', 'premium' ];
```

Comment:
I think we should also remove `business` and `premium` from these lists. Currently the React components don't support those and it actually displays the list of standard plugins. Whether or not you want to simplify the logic based on this change is up to you, but I would imagine it could simplify things.